content/ja/handbook/engineering/devops/plan/index.md

```
@@ -0,0 +1,708 @@
+
+将来のリリースのキャパシティを計画する際、以下の点を考慮します:
+
+1. 次のリリース中のチームの稼働状況（メンバーが休暇を取るか、他の要求がある場合など）。
+1. 現在開発中だが完了していない作業。
+1. グループごとの過去の実績（ウェイト別）。
+
+最初の項目は最大キャパシティとの比較を可能にします。例えば、チームに4人いて、そのうちの1人が月の半分を休暇にとる場合、チームの最大キャパシティの 87.5%（7/8）があると言えます。
+
+2番目の項目は難しく、特にその Issue が他の Issue をブロックしている場合、一度着手した Issue に残っている作業量を過小評価しがちです。私たちは現在、キャリーオーバーした Issue のウェイトを再設定しない（元のウェイトを保持するため）ので、これは現時点では少し曖昧なままです。
+
+3番目の項目はこれまでの進め方を示しています。傾向が下降している場合は、[回顧](#retrospectives) で話し合いましょう。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)